js/src/forum/components/FlagsPage.js

```diff
@@ -12,13 +12,10 @@ export default class FlagsPage extends Page {
 
     app.history.push('flags');
 
-    this.list = new FlagList();
-    this.list.load();
```

Comment:
The `load` call was not removed in https://github.com/flarum/core/pull/2185/files#diff-cccacae14c365a3acf86b9b7aeff3a9a. On purpose?